src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,27 @@
-export const CommentInfo = () => <>Put the comment here</>;
+import './CommentInfo.scss';
+
+function getNamebyEmail(email) {
+  const indexAt = email.indexOf('@');
+
+  const name = email.slice(0, indexAt);
+
+  return name;
+}
+
+export const CommentInfo = ({ comment }) => (
+  <>
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{comment.name}</strong>
```

Comment:
Elements should have empty spaces between them for better readability:

Yours:
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/52002856/0fec9f62-6ea4-4e43-8477-8da395eea751)

With Spaces:
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/52002856/3f4fc491-097a-4b94-be06-84cdd5bfeee5)